virtualizarr/manifests/store.py

```diff
@@ -179,12 +179,14 @@ async def get(
             key, marr.metadata.chunk_key_encoding.separator
         )
 
+        if manifest._paths.shape == (1,) and chunk_indexes == ():
+            # TODO: Kerchunk parsers (e.g., NetCDF3) are returning 1 dimensional arrays for scalar arrays, this should be addressed at the source of the issue rather than as a workaround
+            chunk_indexes = (0,)
```

Comment:
Isn't this going to cause issues when reading from references that were created correctly? And isn't this bugfix unrelated to the serializer change?